WordPress/src/main/java/org/wordpress/android/ui/comments/CommentsListFragment.java

```diff
@@ -104,17 +104,12 @@ public CommentStatus toCommentStatus() {
     public void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         ((WordPress) getActivity().getApplication()).component().inject(this);
-        updateSiteOrFinishActivity(savedInstanceState);
-    }
-
-    @Override
-    public void onStart() {
-        super.onStart();
         mDispatcher.register(this);
+        updateSiteOrFinishActivity(savedInstanceState);
     }
 
     @Override
-    public void onStop() {
+    public void onDestroy() {
         mDispatcher.unregister(this);
         super.onStop();
```

Comment:
:facepalm: Sorry! Not good enough of an excuse, but I am practically sleep walking today :(